core/templates/domain/utilities/language-util.service.spec.ts

```diff
@@ -181,3 +189,25 @@ describe('Language util service', function() {
     }]);
   });
 });
+
+describe('Language Constants', () => {
```

Comment:
[optional] We do have a test for the constants in constants_test.py maybe you can have this there? (constants.py ensures that a few constants are in the correct schema)

Reply:
I moved this test into constants.py. This is my first backend test, so let me know if something looks wrong.